Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'ubuntu/xenial64'
   config.vm.hostname = "#{HOSTNAME}"
+  config.vm.network "forwarded_port", guest: 22, host: 2200, auto_correct: true
```

Comment:
Thanks for detailed explanation!
I'm OK with the change, since the port is not random anymore.